### PR TITLE
Add a get all user shelves method

### PR DESCRIPTION
**Pull request by seankwon**
Added a method to get all shelves from a certain user. Let me know if I need to change anything to make anything cleaner

### CODE REVIEW

File: spec/fixtures/list.xml [hunk 1]
```diff
@@ -0,0 +1,1396 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<GoodreadsResponse>
+   <Request>
+      <authentication>true</authentication>
+      <key><![CDATA[GFPTphT7xVUhrarWQztUtg]]></key>
```
Comment (sosedoff) [inline spec/fixtures/list.xml]: FYI the api key got exposed here.

Comment (seankwon) [inline spec/fixtures/list.xml]: Dont worry, that's their test api key

Comment (seankwon) [inline spec/fixtures/list.xml]: Also, currently working on the travis ci test issue. Trying to see if changing the version num of active support will fix it.

**Comment by seankwon** [comment]
I may need to close and delete this branch because of the data I accidentally compromised. Will open a new pull request and close this one for now, until i purge all history with my accident.

**Pull request closed without merging**